### DISCUSSION TITLE
Feature proposal: Join/merge panes

### DISCUSSION THREAD

**Discussion by tcard** [zed-industries/zed#12115] [Feature Requests]
In VSCode, "View: Join Editor Group With Next Group" does the equivalent of (in Zed terms) moving all open files from a pane to the next pane and then closing the pane.

This allows you to keep working with fewer panes, while keeping all of the current open files and without having to worry about their state of save.

This is a feature I miss while using Zed.

**Comment by JosephTLyons** [comment]
Hey @tcard - I went ahead and created an Issue for you out of this Discussion, since upvotes on Issues is how we track interest in a feature:

- https://github.com/zed-industries/zed/issues/12409

Feel free to upvote with a 👍 reaction!
